bin/git-ignore-io

```diff
@@ -0,0 +1,124 @@
+#/bin/sh
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n"`
```

Comment:
The first method cannot run on my Mac. I found the solution on this [Replace comma with newline in sed](http://stackoverflow.com/questions/10748453/replace-comma-with-newline-in-sed).

Since I'm not the author of gitignore.io, I cannot promise there won't be one. Thus, I use new line as delimiter.